cmsBuild

```diff
@@ -3611,13 +3614,32 @@ def buildSpecs(topLevelPackages):
     if scheduler.brokenJobs:
         sys.exit(1)
 
+def getPackageDependencies(pkg):
+    pkg_list = getPackages(pkg.fullDependencies + [pkg.name])
+    self_description = {}
+    for p in pkg_list:
+        self_description.update({p.name : {"pkgname": p.name, "realversion": p.realVersion,
+                                           "version": p.version,
+                                           "checksum": p.checksum, 
+                                           "patches": [os.path.basename(patch_path) for patch_path in p.patches],
+                                           "dependencies": [d for d in sorted(p.fullDependencies)],
```

Comment:
@mrodozov , may be instead of keeping the expanded full dependnecy its better to keep only the direct one. 
For here, instead of
```
 "dependencies": [d for d in sorted(p.fullDependencies)],
```
may be we can have
```
"dependencies": [d for d in sorted(p.origDependencies)],
"build_dependencies": [d for d in sorted(p.origBuildDependencies)],
```

Reply:
also add
```
  "revision" : p.pkgRevision,
```

Reply:
done, lets run the test again

Reply:
can you add revision too?

Reply:
added